images/example.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Just a note that the pattern in resources uses nesting pretty agressively. I'd like to do that here as well. So there would be two dicts: "image" and "version".

Reply:
Actually it would be `versions` and it would be a list of version dicts. That woudl be best.